native/cocos/platform/mac/modules/SystemWindow.mm

```diff
@@ -43,7 +43,9 @@ of this software and associated engine source code (the "Software"), a limited,
     }
 }
 
-SystemWindow::~SystemWindow() = default;
+SystemWindow::~SystemWindow() {
+    setCursorEnabled(true);
```

Comment:
Why set to true in destructor?

Reply:
If the window exits abnormally in the case of mouse lock, the mouse should be released and displayed.

Reply:
So, it affects system mouse display, not just this program?